bucket/winget-ps.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.4.10173",
+    "description": "A PowerShell Module for the Windows Package Manager Client.",
+    "homepage": "https://github.com/microsoft/winget-cli/tree/master/src/PowerShell/Microsoft.WinGet.Client",
+    "license": "MIT",
+    "suggest": {
+        "PowerShell Core": "pwsh",
+        "WinGet": "winget",
+        "VCRedist2022": "extras/vcredist2022"
```

Comment:
Is this required? It's already present in the winget manifest.

Reply:
The PS  Module requires vcredist, but they would also get vcredist for Winget. So I guess it can be removed as a transitive dependency?

Reply:
Yes